.pre-commit-config.yaml

```diff
@@ -35,3 +35,9 @@ repos:
   hooks:
   - id: conda_envfile_parse
     files: environment.yaml
+# Externally provided executables (so we can use them with editors as well).
```

Comment:
I don't get the remark about editors ;)

Reply:
You can set up your coding environment (vim, VS code, etc) to run clang-format.
For instance, I have it run automatically every time  I save. That way I don't bother formatting anything manually (I don't even make it decent), and run clang-format frequently. It save me some typing.

But if editor's clang-format and pre-commit's clang-format don't have the same version, the output can sometime differ, which is a bit annoying.